xbmc/Util.cpp

```diff
@@ -1754,7 +1754,7 @@ void CUtil::ScanForExternalSubtitles(const std::string& strMovie, std::vector<st
   CFileItemList items;
   vector<std::string> strLookInPaths;
 
-  int flags = DIR_FLAG_NO_FILE_DIRS | DIR_FLAG_READ_CACHE | DIR_FLAG_NO_FILE_INFO;
+  int flags = DIR_FLAG_NO_FILE_DIRS | DIR_FLAG_BYPASS_CACHE | DIR_FLAG_NO_FILE_INFO;
 
```

Comment:
You're sure that dropping will fix the scenario you described where the user enters the directory and then inserts a subtitle? Bypassing it makes sure we`re re-fetching the directory.